src/Ubench.php

```diff
@@ -44,22 +44,33 @@ public function start()
      * Sets end microtime
      *
      * @return void
+     * @throws Exception
      */
     public function end()
     {
+        if (!$this->hasStarted())
+            throw new Exception("You must call start()");
+
         $this->end_time = microtime(true);
         $this->memory_usage = memory_get_usage(true);
     }
 
     /**
      * Returns the elapsed time, readable or not
      *
-     * @param  boolean $readable Whether the result must be human readable
-     * @param  string  $format   The format to display (printf format)
-     * @return string|float
+     * @param bool $raw
+     * @param  string $format The format to display (printf format)
+     * @return float|string
+     * @throws Exception
      */
     public function getTime($raw = false, $format = null)
     {
+        if (!$this->hasStarted())
```

Comment:
Can you follow PSR2 coding style and add some brackets? even for one logic line :)